scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -115,12 +235,13 @@ global.base_stats = { //tempory stats subject to change by anyone that wishes to
 			technology :3,
 			luck :4,
 			skills: {},	
-			gear:{"armour":"Power Armour", "wep1":"Power Sword", "wep2":"Storm Shield"}
+			gear:{"armour":"Power Armour", "wep1":"Power Sword", "wep2":"Storm Shield"},
```

Comment:
didn't we switch to string ids? so Power Armour => power_armour

Reply:
the gear area is not fully implemented but in theory will allow us to pass structs in order to fully re-equip units. so probably however i'm working on a follow up pr that handles implementation of sergeants real squads and stat balancing so will address there.